opentelemetry-exporters-datadog.gemspec

```diff
@@ -16,8 +16,11 @@ Gem::Specification.new do |spec|
   spec.authors     = ['Datadog, Inc.']
   spec.email       = ['dev@datadoghq.com']
 
-  spec.summary     = 'Datadog trace exporter for the OpenTelemetry framework'
+  spec.summary     = '[Deprecated] Datadog trace exporter for the OpenTelemetry framework'
   spec.description = <<-DESC.gsub(/^[\s]+/, '')
+    This exporter has been deprecated. To export your OTLP traces from OpenTelemetry
+    SDK directly to Datadog Agent, please refer to OTLP Ingest in Datadog Agent.\n
+
```

Comment:
@ivoanjo is it possible to have a link in this description?

Reply:
That's a good question. I didn't see anything in the docs, and I tried adding it locally, and I could build the gem. Worst thing that can happen is that it will fail when we try upload the release, but that's harmless and fixable, so I'd say go for it.